docs/category-overview-pages/metrics-streaming-and-replication.md

```diff
@@ -1,3 +1,175 @@
+2. If the parent has been offline for a long time and the child nodes run in db mode `alloc`, you need to plan how you will bring this parent back online. Child nodes in this mode do not have enough retention to backfill the parent and if they connect to it before the other parent, you will end up with missing information on that parent.
+
+The simplest way to solve this, is to block at the firewall all connections to port 19999 from child nodes, but allow connections from the other parent nodes. Once replication finishes for all nodes, you can unblock the connections from child nodes to it.
+
+### I got a parent out of maintenance but it replicates (backfills) missing data slowly. Can I speed it up?
+
+Yes, there is a setting on `netdata.conf` under section `[db]` called `replication threads`. The default value is 1.
+
+Usually each thread is able to replicate about 2-5 million samples per second. We suggest setting this to 5 threads on all parents. Generally do not use too many threads because you are risking congesting the disks and / or the CPU cores available. Keep in mind that the sending parent needs this setting.
+
+There is no need to increase this number on child nodes. Each node has one replication sender, so when hundreds of nodes are replicating to a parent, there are already a lot of senders pushing metrics to it.
+
+### I have multiple active-active parents. Which one is used by Netdata Cloud for queries?
+
+When you have multiple parents available, the one that is further away from the child node is used by Netdata Cloud, unless it does not have the data required.
+
+This works like this: The child has `hops = 0`. Each parent receiving metrics for this child increases the `hops` by 1. So the first parent will have `hops = 1`, the second parent will have `hops = 2` and so on.
+
+Netdata Cloud knows the retention of each parent. So, when it needs data from this child, it first checks the available retention each parent has for it and then it uses the parent with the higher `hops`. If no parent is available and the child node is directly connected to Netdata Cloud, it uses the child.
+
+### Is there a way to balance child nodes to the parent nodes of an active-active cluster?
+
+If you have 2 parent nodes A and B, you can configure them on half the child nodes as A, B and the other half as B, A. The child nodes will connect to the first available (left to right). If both A and B are online, half of the child nodes will connect to A and the other half to B.
+
+Keep in mind however, that if you restart a parent, all the child nodes that were connected to it will automatically reconnect to the other parent. Once this happens, the child nodes will stay connected to it.
```

Comment:
```suggestion
Keep in mind, however, that if you restart a parent, all the child nodes that were connected to it will automatically reconnect to the other parent. Once this happens, the child nodes will stay connected to it.
```